pages/ksphere/kommander/1.2/tutorials/configure-rbac/index.md

```diff
@@ -27,84 +27,204 @@ In this procedure, using these default roles, you will create four different per
 1. Select _Global_ in the header drop-down.
 2. Select _Administration_ > _Access Control_.
-3. Select the _Cluster Roles_ tab.
-   The two default roles that will be used for this persona are:
-
-- _Workspace Admin Role_: because this is a Cluster Role type role, it applies to all target clusters in all workspaces
-- _Kommander Workspace Admin Role_: because this is a Kommander Role type role, it applies to the management cluster. This role grants admin access to the workspace namespace on the management cluster. When a policy is created with this role, a ClusterRole and policy will automatically be created at the global level to allow access to this workspace specifically, instead of all workspaces.
-
+3. Select the _Cluster Roles_ tab. The two roles for this persona are _Workspace Admin Role_ and _Kommander Workspace Admin Role_.
 4. Select the Cluster Policies tab and assign these two roles to the _Workspace Admin_ group.
 
+#### Using kubectl
+
+Assuming `kubectl` is attached to your Kommander cluster and you have a `VirtualGroup` named `workspace-admin`. You can apply this template to link the group to roles:
+
+```yaml
+apiVersion: workspaces.kommander.mesosphere.io/v1alpha1
+kind: VirtualGroupWorkspaceRoleBinding
+metadata:
+  name: workspace-admin-role-binding
+  namespace: <namespace-of-workspacerole-workspace-admin>
+spec:
+  workspaceRoleRef:
+    name: workspace-admin
+  virtualGroupRef:
+    name: workspace-admin
+---
+apiVersion: workspaces.kommander.mesosphere.io/v1alpha1
+kind: VirtualGroupKommanderWorkspaceRoleBinding
+metadata:
+  name: workspace-admin-kommander-role-binding
+  namespace: <namespace-of-kommanderworkspacerole-kommander-workspace-admin>
+spec:
+  kommanderWorkspaceRoleRef:
+    name: kommander-workspace-admin
+  virtualGroupRef:
+    name: workspace-admin
+```
+
 ### Workspace Viewer persona
 
-The _Workspace Viewer_ persona will grant users read-only access to the Default Workspace. This allows users to view all namespaced resources in the workspace namespace on the management cluster, as well as all cluster resources on the workspace’s target clusters.
+The _Workspace Viewer_ persona will grant users in a speific group read access to the Default Workspace. Users in the group can view all namespaced resources in the workspace namespace on the management cluster, as well as all cluster resources on the workspace’s target clusters.
+
+The two roles for this persona are:
+
+- _Workspace View Role_: a `WorkspaceRole` that applies to all target clusters in all workspaces
+- _Kommander Workspace View Role_: a `KommanderWorkspaceRole` that applies to the management cluster. This role grants read access to the workspace namespace on the management cluster. When a `VirtualGroupKommanderWorkspaceRoleBinding` with this role gets created, a `ClusterRole` and `VirtualGroupKommanderClusterRoleBinding` will automatically be created at the global level to allow access to this workspace specifically, instead of all workspaces.
```

Comment:
```suggestion
- _Kommander Workspace View Role_: a `KommanderWorkspaceRole` that applies to the management cluster. This role grants read access to the workspace namespace on the management cluster. When a `VirtualGroupKommanderWorkspaceRoleBinding` with this role gets created, a `ClusterRole` and `VirtualGroupKommanderClusterRoleBinding` create at the global level to allow access to this specific workspace, instead of all workspaces.
```